client/cmd.go

```diff
@@ -147,6 +150,23 @@ func ReadPersistentCommandFlags(clientCtx Context, flagSet *pflag.FlagSet) (Cont
 		}
 	}
 
+	if clientCtx.ClientConn == nil || flagSet.Changed(flags.FlagGRPCNode) {
+		grpcURI, _ := flagSet.GetString(flags.FlagGRPCNode)
+
+		if grpcURI != "" {
+			grpcKeepAlive, _ := flagSet.GetDuration(flags.FlagGRPCKeepAlive)
+			gcl, err := grpc.DialContext(ctx, grpcURI,
+				grpc.WithInsecure(),
+				grpc.WithKeepaliveParams(keepalive.ClientParameters{
+					Time: grpcKeepAlive,
+				}),
+			)
+			if err == nil {
+				clientCtx = clientCtx.WithClientConn(gcl)
+			}
```

Comment:
```suggestion
			if err != nil {
				return clientCtx, err
			}
			clientCtx = clientCtx.WithClientConn(gcl)
```

Reply:
Prefer returning a zero-value Context when the error is non-nil.